libr/core/cmd.c

```diff
@@ -2636,7 +2636,7 @@ static int r_core_cmd_subst_i(RCore *core, char *cmd, char *colon, bool *tmpseek
 			if (ptr[2] == '.') { // "@.."
 				if (ptr[3] == '.') { // "@..."
 					ut64 addr = r_num_tail (core->num, core->offset, ptr + 4);
-					r_core_block_size (core, R_ABS (addr - core->offset));
+					r_core_block_size (core, R_ABS ((int)addr - (int)core->offset));
```

Comment:
It is wrong because you are reducing 64bit integer to 32bit integer on some systems. Please using 64bit integer here.

Reply:
It is OK, because int is the type of the bsize parameter anyway; so I am not causing much additional harm.